CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+    - Check your code works as expected and document any changes to the Lua API.
+
+3. Commit & [push](https://help.github.com/articles/pushing-to-a-remote/) your changes to a new branch (not `master`, one change per branch)
+    - Commit messages should:
+        - Use the present tense
+        - Have a title which begins with a capital letter
+        - Be descriptive. (e.g. no `Update init.lua` or `Fix a problem`)
+        - Have a first line with less than *80 characters* and have a second line that is *empty*
+
+4. Once you are happy with your changes, submit a pull request.
+     - Open the [pull-request form](https://github.com/minetest/minetest/pull/new/master)
+     - Add a short description explaining briefly what you've done (or if it's a work-in-progress - what you need to do)
+
+##### A pull-request is considered merge-able when:
+
+1. It follows the [roadmap](https://forum.minetest.net/viewtopic.php?t=9177) in some way and fits the whole picture of the project.
+2. It works.
+3. It follows the code style for [C/C++](http://dev.minetest.net/Code_style_guidelines) or [Lua](http://dev.minetest.net/Lua_code_style_guidelines).
+4. The code's interfaces are well designed, regardless of other aspects that might need more work in the future.
+5. It uses protocols and formats which include the required compatibility.
+
+## Issues
+
+If you experience an issue, we would like to know the details - especially when a stable release is on the way.
+
+1. Do a quick search on GitHub to check if the issue has already been reported.
+2. Is it an issue with the Minetest *engine*? If not, report it [elsewhere](http://www.minetest.net/development/#reporting-issues).
+3. [Open an issue](https://github.com/minetest/minetest/issues/new) and describe the issue you are having - you could include:
+     - Error logs (check the bottom of the `debug.txt` file)
+     - Screenshots
+     - Ways you have tried to solve the issue, and whether they worked or not
+     - Your Minetest version and the content (subgames, mods or texture packs) you have installed
+     - Your platform (e.g. Windows 10 or Ubuntu 15.04 x64)
+
+After reporting you should aim to answer questions or clarifications as this helps pinpoint the cause of the issue (if you don't do this your issue may be closed after 1 month).
+
+## Feature requests
+
+Feature requests are welcome but take a moment to see if your idea follows the [roadmap](https://forum.minetest.net/viewtopic.php?t=9177) in some way and fits the whole picture of the project. You should provide a clear explanation with as much detail as possible.
+
+## Translations
+
+Translations of Minetest are performed using Weblate. You can access the project page  with a list of current languages [here](https://hosted.weblate.org/projects/minetest/minetest/).
+
+[Contact a core developer](http://www.minetest.net/irc/) if you'd like to translate into a language that isn't there.
```

Comment:
This isn't accurate anymore, you can add new languages without approval of a core dev.